### ISSUE TITLE
[FEATURE] Combine stats from multiple browsers (or auto-sync)

### ISSUE THREAD

**Issue by rolltidehero** [Stigmatoz/web-activity-time-tracker#100]
Would it be possible to add a feature to restore a backup from a different browser\workstation and have it combine with the currents stats instead of overwriting them? 

That would be a good start. Then eventually, an auto-sync option would be amazing!

I work with several different browsers on several different workstations, so something like this would be a huge help for me.

Currently, I am using a tool to combine the exported data and then restore it. It's not a user-friendly task though.

Thanks so much for your work on this tool. It blows all of the other trackers out of the water!

**Comment by sheepzh** [comment]
Maybe you want this

![image](https://github.com/Stigmatoz/web-activity-time-tracker/assets/19428473/8a627ad4-acee-453c-add0-9231b221b3f1)

![image](https://github.com/Stigmatoz/web-activity-time-tracker/assets/19428473/6bea4749-af39-44fa-99e1-041e60530888)

**Comment by rolltidehero** [comment]
> Maybe you want this
> 
> ![image](https://user-images.githubusercontent.com/19428473/246003347-8a627ad4-acee-453c-add0-9231b221b3f1.png)
> 
> ![image](https://user-images.githubusercontent.com/19428473/246003594-6bea4749-af39-44fa-99e1-041e60530888.png)

Yea... Is that a different extension because I don't have that page?

**Comment by sheepzh** [comment]
Yes, you can install it [here](https://chrome.google.com/webstore/detail/time-tracker/dkdhhcbjijekmneelocdllcldcpmekmm).

**Comment by rolltidehero** [comment]
> Yes, you can install it [here](https://chrome.google.com/webstore/detail/time-tracker/dkdhhcbjijekmneelocdllcldcpmekmm).

Ahh yea, I've tried that and unfortunately it isn't free if you want to track websites. Thanks though.
